packages/dm-core-plugins/src/responsive_grid/ResponsiveGridPlugin.tsx

```diff
@@ -0,0 +1,54 @@
+import { IUIPlugin, Stack, ViewCreator } from '@development-framework/dm-core'
+import { Typography } from '@equinor/eds-core-react'
+import React from 'react'
+import { Col, Container, Row } from 'react-grid-system'
+import { TCol, TGridPluginConfig, TRow } from './types'
+
+const defaultConfig: TGridPluginConfig = {
+  rows: [],
+}
+
+export const ResponsiveGridPlugin = (
+  props: IUIPlugin & { config: TGridPluginConfig }
+): React.ReactElement => {
+  const { config, idReference, type, onSubmit, onChange } = props
+
+  const internalConfig: TGridPluginConfig = {
+    ...defaultConfig,
+    ...config,
+  }
+
+  return (
+    <Stack
+      style={{
+        width: '100%',
+        height: '100%',
+        display: 'flex',
+        maxWidth: 'max-content',
+      }}
+    >
+      <Container>
+        {internalConfig.rows.map((row: TRow) => {
+          const columns = row.columns.map((col: TCol) => {
+            return (
+              <Col {...col.size}>
+                <div style={{ marginBottom: '40px' }}>
```

Comment:
We should be using grid gap and I feel like default should be much smaller, maybe 1rem/16px? And unless <Col /> can only have one child or it's required by react-grid-system I don't think there's a need for a div inside